javascript/event-recommender-part2.md

```diff
@@ -23,11 +23,11 @@ Now that you've finished Part 1 and your Event Recommender class is complete, yo
 
 4. You are required to *deploy* your completed project using Netlify or Github Pages. Send the links to your *Github repo* and *live site url*.
 
-5. You are required to have a top-notch README.md with screenshots of your project in your Github project repo.
+5. You are required to have a top-notch README.md with screenshots of your project in your Github project repo. (Here is an example of a great README.md: https://gist.github.com/PurpleBooth/109311bb0361f32d87a2)
 
 ### UI Instructions
 
-1. Create HTML and CSS files. In your HTML file, add a script tag that includes your existing JavaScript file that defines your EventRecommender and other classes. Then create a new file, `jQuery-scripts.js`, and add it via a script tag after your other script tag.
+1. Create HTML and CSS files. In your HTML file, add a script tag that includes your existing JavaScript file that defines your EventRecommender and other classes. Then create a NEW file, `jQuery-scripts.js`, and add it via a script tag after your other script tag. (It is important that your classes and jQuery are seperate Javascript files, because you will only run Jasmine test on your classes, not your jQuery code.)
```

Comment:
what's the point on having `NEW` in uppercase?

Reply:
Strong emphasis, but not necessary.

Reply:
Hi @jnnnle, I’d have to disagree with this statement. There’s no reason that jQuery can’t be in any JS file, and it does not interfere with testing - this file’s code just has a different purpose, so separating them keeps things organized/modular.

Also, consider changing NEW to *new* or **new**, as capitalizing the whole word causes those using screen readers to be read one letter at a time, since it’s interpreted to be an acronym.

Reply:
I will change NEW to <b>new</b>.

Consider that when one includes their jquery in your js file and when they run the Jasmine tests, $ is not recognized by Jasmine. The front end and backend do not communicate with one another well, and I ran into this error that I tried to resolve with Leah; this was our simplest solution. So if we isolate the problem, students can run their jasmine test as expected.